pareto/utilities/get_data.py

```diff
@@ -351,15 +352,18 @@ def get_display_units(input_sheet_name_list, user_units):
         "CST": "",
         "CCT": "",
         "CKT": "",
+        "Elevation": pyunits.foot,
```

Comment:
```suggestion
        "Elevation": "foot",
```
All of the other entries in this dictionary are strings, so I think this should probably be a string as well. (If you commit this change then you can also remove the line `from pyomo.environ import units as pyunits` from the file).

Reply:
updated the units